src/QueryBuilder/searchModifiers.ts

```
@@ -0,0 +1,27 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+export enum SearchModifier {
+    None = 'none',
+    Error = 'error',
```

Where is this used? It's probably better to throw an exception instead

---

I replaced this enum with the return object you suggested in a different message.